sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AlertTriggering/BoundaryMeasureType.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.AI.MetricsAdvisor.Models
+{
+    /// <summary>
+    /// Used as part of a <see cref="MetricBoundaryCondition"/>. Specifies which measure should be
+    /// used when checking boundaries. Defaults to <see cref="BoundaryMeasureType.Value"/>.
+    /// </summary>
+    [CodeGenModel("ValueType")]
+    public readonly partial struct BoundaryMeasureType
+    {
+        /// <summary>
+        /// The value of metric is used as it is.
```

Comment:
nit: it could be me and English but feels like it should be `of a metric` or `of the metric`

Reply:
You're right. Sounds awkward. Updated to "of the metric".